pkg/kab/relocate_test.go

```diff
@@ -17,61 +17,84 @@
 package kab_test
 
 import (
+	"cnab-k8s-installer-base/pkg/apis/kab/v1alpha1"
+	"cnab-k8s-installer-base/pkg/kab"
+	"cnab-k8s-installer-base/pkg/kab/vendor_mocks"
+	"cnab-k8s-installer-base/pkg/registry/mocks"
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pivotal/image-relocation/pkg/image"
+	"github.com/stretchr/testify/mock"
 )
 
 var _ = Describe("RelocateManifest", func() {
 
-	//var (
-	//	manifest *v1alpha1.Manifest
-	//	destRepo string
-	//	reloMap  map[string]string
-	//	err      error
-	//)
-	//
-	//BeforeEach(func() {
-	//	manifest = &v1alpha1.Manifest{
-	//		Spec: v1alpha1.KabSpec{
-	//			Resources: []v1alpha1.KabResource{
-	//				{
-	//					Content: "spec:\n" +
-	//						"  template:\n" +
-	//						"    spec:\n" +
-	//						"      containers:\n" +
-	//						"      - -builder\n" +
-	//						"      - cluster\n" +
-	//						"      - INFO\n" +
-	//						"      image: gcr.io/knative-releases/x/y\n" +
-	//						"      name: build-webhook\n",
-	//				},
-	//				{
-	//					Content: "spec:\n" +
-	//						"  containers:\n" +
-	//						"  - image: mysql:5.6\n" +
-	//						"  name: mysql",
-	//				},
-	//			},
-	//		},
-	//	}
-	//	destRepo = "my.private.repo"
-	//})
-	//
-	//Context("when the manifest has Spec.Resource.Content", func() {
-	//	It("repository relocation is successful", func() {
-	//		reloMap, err = kab.Relocate(manifest, destRepo)
-	//		Expect(err).To(BeNil())
-	//
-	//		for i := range manifest.Spec.Resources {
-	//			Expect(manifest.Spec.Resources[i].Content).NotTo(ContainSubstring("docker.io"))
-	//			Expect(manifest.Spec.Resources[i].Content).NotTo(ContainSubstring("gcr.io"))
-	//			Expect(manifest.Spec.Resources[i].Content).To(ContainSubstring(destRepo))
-	//		}
-	//
-	//		Expect(reloMap).Should(HaveLen(2))
-	//		Expect(reloMap).Should(HaveKeyWithValue("mysql:5.6", "my.private.repo/mysql:5.6"))
-	//		Expect(reloMap).Should(HaveKeyWithValue("gcr.io/knative-releases/x/y", "my.private.repo/knative-releases-x-y"))
-	//
-	//	})
-	//})
+	var (
+		client             *kab.Client
+		kubeClient         *vendor_mocks.Interface
+		mockRegistryClient *mockregistry.Client
+		manifest           *v1alpha1.Manifest
+		destRepo           string
+		err                error
+	)
+
+	BeforeEach(func() {
+		kubeClient = new(vendor_mocks.Interface)
+		mockRegistryClient = new(mockregistry.Client)
+
+		client = kab.NewKnbClient(kubeClient, nil, nil, mockRegistryClient, nil)
+	})
+
+	BeforeEach(func() {
+		manifest = &v1alpha1.Manifest{
+			Spec: v1alpha1.KabSpec{
+				Resources: []v1alpha1.KabResource{
+					{
+						Content: "spec:\n" +
```

Comment:
Would probably be easier to read if you used `` comment delimiters and real newlines.